examples/classification/plot_lda_oas.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
Would it be possible to update https://scikit-learn.org/stable/auto_examples/classification/plot_lda.html#sphx-glr-auto-examples-classification-plot-lda-py instead of creating a new example?